helm/cluster-api-bootstrap-provider-kubeadm/templates/_resource.tpl

```diff
@@ -26,7 +27,7 @@ See https://github.com/giantswarm/cluster-api-app/blob/master/helm/cluster-api/t
 Issue: https://github.com/giantswarm/giantswarm/issues/19415
 */}}
 {{- define "resource.webhook.name" -}}
-{{- if eq $.Chart.Name $.Release.Name }}cluster-api-bootstrap{{ else }}{{ include "resource.default.name" . }}{{ end }}
+{{- if eq $.Chart.Name $.Release.Name }}cluster-api-bootstrap{{ else }}{{ include "resource.default.name" . }}-webhook{{ end }}
```

Comment:
We need a unique name for the validation webhook otherwise the app in the release CR will fail.

When deployed via an app collection we use the name in the CRD.